https://issues.redhat.com/browse/ACM-25215--software catalog replaces… (#8420)
open-cluster-management/rhacm-docs: install/install_disconnected.adoc

. Install the {acm-short} operator and hub cluster.

+

> After you have configured {olm} and {ocp-short} as previously described, you can install {acm-short} by using either the {ocp-catalog} console or a command-line interface. Follow the same guidance described in the xref:../install/install_connected.adoc#installing-while-connected-online[Installing while connected online] topic.

Reviewer: Vale doesn't like us using "previously" it recommends "earlier" but honestly, I don't think we need this. What do you think of removing it? 

Also, I changed the last sentence to simply be structured more similarly to how we typically introduce xrefs. Simply for consistency, nothing was bad with the original :) 

```suggestion
After you have configured {olm} and {ocp-short}, you can install {acm-short} by using the {ocp-catalog} console or the command-line interface. For directions, see the xref:../install/install_connected.adoc#installing-while-connected-online[Installing while connected online] topic.
```

The `mce-subscription-spec` annotation is required because {mce-short} is automatically installed during the {acm-short} installation. If you are creating the resource with a CLI, include the `mce-subscription-spec` annotation in the YAML that you apply with the `oc apply` command to create the `MultiClusterHub` resource.

+

> If you create the resource by using the console, switch to the _YAML view_ and insert the annotation as previously displayed. *Important:* There is no field in the OperatorHub console for the annotation in the _Field view_ panel to create the `MultiClusterHub` resource.

Reviewer: I thought we had a conref to refer to the console, but I checked and didn't see one, lol.

Author: no, it's pretty basic, just "console"

Reviewer 2: I remember us choosing to italicize panel names, but it seems like we should be using double quotes when the panel name is sentence-case: https://www.ibm.com/docs/en/ibm-style?topic=format-highlighting (search for "Panel")

If it is headline style capitalization, no highlighting. This is minor and something we might need to talk about as a team. I am fine with you keeping it as is, just wanted to bring it to your attention
